### DISCUSSION TITLE
Ubuntu / Windows settings to run Miner! How to change the filename Directory

### DISCUSSION THREAD

**Discussion by PaulCosteFi8** [catchthatrabbit/coreminer#41] [Operating System]
Hi,

I am so far that I have downloaded the entire zip file have done all the steps as described in the procedure, have downloaded being on Windows the Ubuntu the described version have activated as well the virtual machine and all that but in the end, I can't find how to change the name of the directory to start the miner and maybe you could help where do I do that in the Linux or in the Command prompt of Windows itself !?
![photo_2022-05-24_13-49-38](https://user-images.githubusercontent.com/42944733/170019399-78168051-fff6-4543-bbc6-1001275525a4.jpg)

**Comment by rastislavcore** [comment]
Feel free open your request here: https://github.com/catchthatrabbit/coreminer/issues/new/choose

**Reply by PaulCosteFi8** [reply]
Thank you  Rasty, 

I have solved it.

**Reply by glennsheary** [reply]
how did you solve it?
